packages/dm-core-plugins/src/form/fields/BinaryField.tsx

```diff
@@ -14,9 +15,9 @@ const getTarget = (initialValue: any) => {
   const { idReference } = useRegistryContext()
   if (initialValue['address'].includes('blob')) {
     const address = initialValue['address'].replace('blob://', '')
-    return address.split('/', 2)
+    return splitAddress(address).dataSource
```

Comment:
getTarget should return an array of length 2.

Reply:
right 😶 🤦